src/joypad_accessory.c

```diff
@@ -729,12 +740,19 @@ void joypad_accessory_xfer_async(
 }
 
 joypad_accessory_error_t joypad_accessory_xfer(
-    joypad_port_t port, 
+    joypad_port_t port,
     joypad_accessory_xfer_t xfer,
-    uint16_t start_addr, 
-    void *dst,
-    size_t len)
+    uint16_t start_addr,
+    void *data,
+    size_t nbytes)
 {
+    if (sys_bbplayer()) {
+        switch( bb_save_xfer(port, xfer, start_addr, data, nbytes) ) {
```

Comment:
Same issue for `xfer` here:

```C
bb_save_xfer_t = (xfer ==  BB_SAVE_XFER_READ) ? JOYPAD_ACCESSORY_XFER_READ : JOYPAD_ACCESSORY_XFER_WRITE;
```

Reply:
Fixed in https://github.com/DragonMinded/libdragon/pull/680/commits/ecd5a52bedf72b5d45c5b10402ddc09d78f760b6